students/10th/ohogeun/js/login.js

```diff
@@ -4,10 +4,6 @@ loginBox.addEventListener('keyup', function () {
     const button = document.getElementsByClassName('loginButton')[0];
     const idBox = document.getElementsByClassName("loginId")[0].value;
     const pwBox = document.getElementsByClassName("loginPw")[0].value;
-
-    if(idBox && pwBox){
-        button.style.backgroundColor = "blue";
-    }else if (!(idBox && pwBox)) {
-        button.style.backgroundColor = "#B2DFFC";
-    }
+    let buttonColor = button.style.backgroundColor;
+    (idBox && pwBox) ? button.style.backgroundColor = "blue" : button.style.backgroundColor = "#B2DFFC"
```

Comment:
`button.style.backgroundColor` 의 값만 변하는 것이기 때문에 값 부분에만 삼항연산자를 적용하시면 됩니다~
```suggestion
    button.style.backgroundColor =  (idBox && pwBox) ? "blue" : "#B2DFFC"
```

Reply:
넵